modules/pam_faillock/main.c

```diff
@@ -129,17 +128,14 @@ do_user(struct options *opts, const char *user)
 		}
 	}
 	if (opts->reset) {
-#ifdef HAVE_LIBAUDIT
-		int audit_fd;
-#endif
-
 		while ((rv=ftruncate(fd, 0)) == -1 && errno == EINTR);
 		if (rv == -1) {
 			fprintf(stderr, "%s: Error clearing the tally file for %s:",
 				opts->progname, user);
 			perror(NULL);
 #ifdef HAVE_LIBAUDIT
 		}
+		int audit_fd;
```

Comment:
We do not allow variable declarations in the middle of block. This has to stay as it was.

Reply:
ok, however in my view can remove one macro